sbin/jtreg_prep.sh

```diff
@@ -95,7 +95,7 @@ fi
 if [ ! -d "${JAVA_DESTINATION}" ]; then
   mkdir -p "${JAVA_DESTINATION}"
 fi
-cd "${JAVA_DESTINATION}"
+cd "${JAVA_DESTINATION}"  || exit
```

Comment:
Convention has changed on this line and below, "${VAR}" same as ${VAR} or "$VAR"